src/xor_name.rs

```diff
@@ -298,28 +301,31 @@ mod tests {
 
     #[test]
     fn xor_name_equal_assertion() {
-        let type1: XorName = rand::random();
+        let mut rng = rand::thread_rng();
+        let type1: XorName = rng.gen::<XorName>();
         let type1_clone = type1;
-        let type2: XorName = rand::random();
+        let type2: XorName = rng.gen::<XorName>();
```

Comment:
do we need to declare the type twice? won't `rng.gen();` work?

Reply:
I think this test is perhaps useless. It seems to only test `.clone` works and generating 2 random XorNames will be guaranteed different (which it won't). Personally I would remove this test.